### ISSUE TITLE
duplication and deletion at same location?

### ISSUE THREAD

**Issue by jmodlis** [fritzsedlazeck/Sniffles#106]
Hi,

I am trying to understand this output. It seems to be saying there is a deletion and duplication in the same location?  How would this work?  Based on mapping data, we had thought it possible that there was a small inversion here, or something else strange going on.

Thanks!
Jen
```
#Chrom  start   stop    chrom2  start2  stop2   variant_name/ID score (smaller is better)       strand1 strand2 type    number_of_split_reads   best_chr1       best_start      best_chr2       best_stop       predicted_length        FILTER
II      12938537        12938537        II      13964873        13964873        4138    -1      +       -       DEL     13      II      12938540        II      13965086        1026546 IMPRECISE
II      12939809        12939809        II      13967893        13967893        4139    -1      -       +       DUP     11      II      12939813        II      13967893        1028080 PRECISE
```

**Comment by fritzsedlazeck** [comment]
Oh, is your organism diploid?
Cheers
Fritz

**Comment by jmodlis** [comment]
Ok, that is kind of what I was thinking, but then I plotted depth of coverage of the reads over the reference and against an assembly I created, and didn't see anything strange, and I also did a dotplot of the assembly versus reference and saw nothing strange.  I will try your vcf suggestion and let you know what I find!  Thanks!

**Comment by fritzsedlazeck** [comment]
ah most of the time these things are displayed as split reads. If you go for IGV you have sometimes to set the preferences in order to be able to see them.
Hope that helps. Keep me posted
Thanks
Fritz

**Comment by jmodlis** [comment]
Hi Fritz,

Thanks for your help...So here are the results with --genotype --cluster. It looks like the deletion is on one strand and the duplication is on the other strand.  I'm trying to wrap my head around what this means ? And why would reads preferentially map to one strand and not the other?

Thanks,
Jen

```II      12938540        4138    N       <DEL>   .       PASS IMPRECISE;SVMETHOD=Snifflesv1.0.8;CHR2=II;END=13965086;ZMW=13;STD_quant_start=4.472136;STD_quant_stop=263.342742;Kurtosis_quant_start=3.143768;Kurtosis_quant_stop=-0.968467;SVTYPE=DEL;SUPTYPE=SR;SVLEN=1026546;STRANDS=+-;RE=13;AF=1  GT:DR:DV        1/1:0:13```

```II      12939813        4139    N       <DUP>   .       PASS    PRECISE;SVMETHOD=Snifflesv1.0.8;CHR2=II;END=13967893;ZMW=11;STD_quant_start=4.658326;STD_quant_stop=1.140175;Kurtosis_quant_start=6.700892;Kurtosis_quant_stop=6.999586;SVTYPE=DUP;SUPTYPE=SR;SVLEN=1028080;STRANDS=-+;RE=11;AF=1       GT:DR:DV        1/1:0:11```

**Comment by fritzsedlazeck** [comment]
Hi Jen,
Yes this is really hard... 
What you could also do is to see if that persists also with a different aligner. So if you used ngmlr, try minimap2 or the other way around. Just make sure that minimap2 reports the MD tag.
Cheers
Fritz

**Comment by jmodlis** [comment]
Hi Fritz,

So I ran minimap2 and the output files are empty, aside from the headers.  It is not an MD tag issue, because I ran minimap2 with the --MD parameter (not the first time, but I did the second time ;).  Have you see this before? Previously I was running ngmlr.

When I look at the reads in IGV, it looks like there is a gap in the alignments at the start and stop points, but I can't tell what is going on as far as strandedness.

Thanks!
Jen

**Comment by fritzsedlazeck** [comment]
Hi Jen,
I haven't used minimap2 often ( for obvious reasons ;)).
May I ask what organism it is?
Cheers
Fritz

**Comment by jmodlis** [comment]
Hi Fritz,

Sorry for the delayed reply.  I never truly got to the bottom of this.  I tried alternative assemblers, the Pacbio smrtlink SV caller, and nothing gave me an insightful answer.  I did examine the alternate contigs that falcon/hgap4 outputs, and based on those and some dot plots, I think it might have been a section of expanded repeats that was also heterozygous and repeated elsewhere in the chromosome.

Thanks for your help!
Jen
